app/src/logout.js

```diff
@@ -0,0 +1,15 @@
+import "./login.css";
+
+async function logoutUser() {
+  await fetch("http://localhost:1000/logout", {
+    method: "POST",
+  }).then((response) => response.json());
+}
+
+function Logout() {
+  logoutUser();
+  window.open("/login", "_self");
+  sessionStorage.removeItem("token");
```

Comment:
These two should happen only when we have received the response from the logout API, or in other words only when the API has resolved.